gears/pennbbl/flaudit.json

```diff
@@ -0,0 +1,35 @@
+{
+	"name": "flaudit",
+	"label": "Flaudit: Flywheel Audit",
+	"description": "Audit your Flywheel project for sequence completeness, BIDS curation summaries, gear and job runs, workflow completeness, and more.",
+	"version": "0.0.5_0.1.6",
+  "author": "Tinashe Michael Tapera",
+	"maintainer": "Tinashe Michael Tapera",
+	"cite": "",
+	"license": "BSD-3-Clause",
+	"source": "",
+	"url": "https://https://fw-heudiconv.readthedocs.io/en/latest/",
```

Comment:
This has extra "https://" but shouldn't this be https://github.com/PennLINC/flaudit?

Reply:
Thanks @AndysWorth !